packages/server/tests/localAcceptance.env

```diff
@@ -1,6 +1,7 @@
 HEDERA_NETWORK={"127.0.0.1:50211":"0.0.3"}
-OPERATOR_ID_MAIN=0.0.2
-OPERATOR_KEY_MAIN=302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137
+# Account from local node. This is the account that will be used to pay for transactions.
+OPERATOR_ID_MAIN=0.0.1022
+OPERATOR_KEY_MAIN=302e020100300506032b657004220420a608e2130a0a3cb34f86e757303c862bee353d9ab77ba4387ec084f881d420d4
```

Comment:
After local-node finishes running, I could find the account 0.0.1022 in the accounts list, and there seems to be only HEX private keys, not DER. So I'm curious about how you got this DER key?

Reply:
Good point. 
Also don't we now support the hex format for keys? If so let's use that instead

Reply:
Done.  Now using ECDSA key in HEX format.